README.md

```diff
@@ -50,25 +51,22 @@
 </details>
 
 
-
-
-
 ### Key Features <a name="key-features"></a>
 
-
--
+- **[Display a list of stocks and services   ]**
+- **[Filter the companies by typing in the search bar  ]**
+- **[Display individual information about a selected company  ]**
+- **[Backward arrow to return to the previous page ]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-[Live demo Link ](https://dainty-sunshine-0d74b1.netlify.app/)
+[Live demo Link ](https://dev--inquisitive-mandazi-5eae9a.netlify.app/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Video desctiption <a name="video"></a>
-![big-tv-tv]()
-
-[Video Link ]()
+[Video Link ](https://www.loom.com/share/e11da9d2a74c4611a33c8194091125b7)
```

Comment:
- [ ] [REQUIRED] Great presentation @MICHAELKITH 👍🏽 . However, as stated by the project requirement, it is expected that you a piece of the code that you are proud of making. This will convince me as a reviewer that you actually build this application by yourself.
- [ ] [OPTIONAL] I will recommend that you put on your camera when making a project presentation. This tends to be more professional and it will prepare you ahead of more project presentations when you eventually start to work as a developer in a company of your choice.